README.md

```diff
@@ -130,6 +134,15 @@ The `version` is read from the `AUTOMETRICS_VERSION` environment variable, and t
 
 This follows the method outlined in [Exposing the software version to Prometheus](https://www.robustperception.io/exposing-the-software-version-to-prometheus/).
 
+## Exemplars
+
+> **NOTE** - As of writing, aren't supported by the default tracker (`AUTOMETRICS_TRACKER=OPEN_TELEMETRY`).
```

Comment:
good eye! indeed, the word was missing - fixed now

Reply:
i added the note above for consistency with the previous paragraph, it would look like this:
<img width="1214" alt="Screenshot 2023-06-14 at 10 59 07" src="https://github.com/autometrics-dev/autometrics-py/assets/90313638/3042d96c-86f8-420b-bb4c-d173c7137cd0">

i think since it's quoted people can just look below, but the quoted part is quite big. if you think it's not obvious i will move the note below

Reply:
It makes more sense to me to explain first and then mention what the caveats are especially if people aren't super familiar with what exemplars are even if you can skip, notes are typically important so for me (personally) I would still try to read it first., but that's just anecdotal (and also I don't feel that strongly about it) so i'm fine with leaving it the way you have it now.

Reply:
i ended up moving a bunch of text from the quoted part below, i think this should be fine now
<img width="1122" alt="Screenshot 2023-06-14 at 14 19 30" src="https://github.com/autometrics-dev/autometrics-py/assets/90313638/0282632c-e74c-4c1a-9bb7-addda4728126">